large_files.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I'm guessing this file isn't meant to be here.

Reply:
No idea how it ended up there. Your guess is correct, removed.